schematic/manifest/commands.py

```diff
@@ -106,6 +106,9 @@ def get_manifest(ctx, title, data_type, jsonld, dataset_id, sheet_url,
         logger.info(
             f"Find the manifest template using this CSV file path: {output_csv}"
         )
-        result.to_csv(output_csv, index=False)
+        if output_csv is None:
+            click.echo(result.to_csv(index=False))
+        else:
```

Comment:
Good catch! In fact, because the CSV filename is auto-generated if `output_csv is None`, then there's no case when the CSV will be printed to stdout. Since I don't think this is an important use case, I'm just going to simplify the logic as follows: 

```
    if sheet_url:
        logger.info("Find the manifest template using this Google Sheet URL:")
        click.echo(result)
    elif isinstance(result, pd.DataFrame):
        if output_csv is None:
            prefix, _ = os.path.splitext(jsonld)
            prefix_root, prefix_ext = os.path.splitext(prefix)
            if prefix_ext == ".model":
                prefix = prefix_root
            output_csv = f"{prefix}.{data_type}.manifest.csv"

        logger.info(
            f"Find the manifest template using this CSV file path: {output_csv}"
        )

        result.to_csv(output_csv, index=False)
```